ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/HttpTimeoutTest.kt

```diff
@@ -504,7 +504,7 @@ class HttpTimeoutTest : ClientLoader() {
     }
 
     @Test
-    fun testSocketTimeoutWriteFailOnWrite() = clientTests(listOf("Js", "Android", "native:CIO")) {
+    fun testSocketTimeoutWriteFailOnWrite() = clientTests(listOf("Js", "Android", "native:CIO", "Java")) {
```

Comment:
should we drop this test?

Reply:
why? it works for engines that use `socketTimeout` from config